docs/components/text/README.md

```diff
@@ -0,0 +1,1288 @@
+
+### Paragraphs
+
+To ensure that usage of this component complies with accessibility guidelines:
+
+  - Text container does not exceed 634px or line length does not exceed more than 75 characters
+  - Minimize use of reversed-out body copy (light copy on a dark background) because it’s harder to read
+  - When possible, use the first sentence as an introduction to the paragraph. With screen readers, users can listen to the first sentence and then jump to the next paragraph
+  - Break long pages into shorter sections by organizing content into well-defined groups or chunks
+
+This component has compliance with following WebAIM’s accessibility guidelines:
+
+  - [WCAG SC 1.4.8: Visual presentation](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-visual-presentation): Cedar Design System text component provides for spacing for:
+    - Within paragraphs, line spacing is at least 1.5 times font height
+
+# Implementation: 
+
+ To work as intended, Tokens, Utility classes and Component require both `cdr-reset.css` and access to the REI Brand fonts. for more information on how to access these resources visit the [developer getting started](https://rei.github.io/rei-cedar-docs/getting-started/as-a-developer/?active-link=setting-up-projects) page.
+
+ The global text default `cdr-text-utility-300` is defined in each of the available cdr-container classes. Make sure that all content is contained in one of the [available container classes](http://localhost:8080/layout/responsive/#standard-vs-fluid-container)
+
+The cedar text css selectors provide no spacing values other than letter-spacing and line-height. 
+
+For proper spacing between blocks of text, use the [cedar spacing options](../..//layout/spacing/)
+
+The cedar text options can be utilized by your application via [token mixins](https://www.npmjs.com/package/@rei/cdr-tokens), [utility classes](https://www.npmjs.com/package/@rei/cedar) or through the [cdr-text component](https://www.npmjs.com/package/@rei/cedar).
+
+## Text Token mixins
+
+Cedar uses design tokens to store typographic attributes that represent the fundamental decisions of Cedar’s visual language.
+
+For a complete list of Cedar design tokens, visit the [Cedar Tokens](https://rei.github.io/rei-cedar-tokens/) page.
+
+## Utility classes
+
+`cdr-text` styles are available as a standalone css classes for users unable or preferring not to add custom styles to their applications, and who are unable to use the `cdr-text` vue.js component.
+
+
+For more information on importing these styles and how to use them, visit the [Developer getting started](./getting-started/as-a-developer/) page.
+
+### Usage
+
+The `cdr-text` root class sets all value pairs to `inherit`. When using these classes,  ensure at least the parent class assigns the `cdr-text` root style.
+
+The utility classes follow [BEM](http://getbem.com/)conventions and have two dashes between `cdr-text` and its modifier `body-300`
```

Comment:
The utility classes have two dashes between `cdr-text` and its modifier `body-300`

not actually BEM, just a separator